packages/atomic/src/components/facets/atomic-facet/atomic-facet.tsx

```diff
@@ -178,6 +178,28 @@ export class AtomicFacet implements InitializableComponent, BaseFacet<Facet> {
    */
   @MapProp() @Prop() public dependsOn: Record<string, string> = {};
 
+  /**
+   * Specifies an explicit list of `allowedValues` in the Search API request, separated by commas.
+   *
+   * If you specify a list of values for this option, the facet uses only these values (if they are available in
+   * the current result set).
+   *
+   * Example:
+   *
+   * The following facet only uses the `Contact`, `Account`, and `File` values of the `objecttype` field. Even if the
+   * current result set contains other `objecttype` values, such as `Message`, or `Product`, the facet does not use
+   * those other values.
+   *
+   * ```html
+   * <atomic-facet field="objecttype" allowed-values="Contact,Account,File"></div>
+   * ```
+   *
+   * The maximum amount of allowed values is 25.
+   *
+   * Default value is `undefined`, and the facet uses all available values for its `field` in the current result set.
+   */
+  @Prop() public allowedValues?: string;
```

Comment:
I recall commenting on this but I don't remember where, that I wonder if we should support `string[]` props (declared similarly to MapProp) so that in React and Angular we can pass an array directly.

What do you think?

Reply:
We could do that, yes. But it seems out of scope of this PR. There's many props in Atomic like this, so we would need to go at it in one go, not only for this one.